src/js/components/Data/filter.js

```diff
@@ -60,10 +60,16 @@ export const filter = (data, view, properties) => {
           );
 
         // options case
-        if (Array.isArray(filterValue))
-          return Array.isArray(value)
-            ? !value.some((v) => filterValue.includes(v))
-            : !filterValue.includes(value);
+        if (Array.isArray(filterValue)) {
+          if (Array.isArray(value)) {
+            return !filterValue.some((f) =>
+              typeof f === 'object'
+                ? value.includes(f.value) // from {label: ___, value: ___}
```

Comment:
It looks like previous we were doing a "filterValue.includes(value)" pattern and now we're doing "value.includes(filter)" -- can you explain why we made this switch? Was that a bug in the initial implementation?

Reply:
Connected over call - `value.includes(filter)` is used in this case because we iterate over `filterValue` and we use `typeof f` to determine if each `f` is in the form of an object `{label: ___, value: __}`. If so, we want to check whether or not `value` includes `f.value`. Previously, there was no check for `typeof f`, which resulted in incorrect behavior when each `value` was a `{label: __, value: __}` object.